src/satosa/micro_services/ldap_attribute_store.py

```diff
@@ -369,58 +369,45 @@ def process(self, context, data):
         the input context.
         """
-        self.context = context
-        state = context.state
-
-        # Find the entityID for the SP that initiated the flow.
-        try:
-            sp_entity_id = context.state.state_dict["SATOSA_BASE"]["requester"]
-        except KeyError as err:
-            msg = "Unable to determine the entityID for the SP requester"
-            satosa_logging(logger, logging.ERROR, msg, state)
-            return super().process(context, data)
-
-        msg = "entityID for the SP requester is {}".format(sp_entity_id)
-        satosa_logging(logger, logging.DEBUG, msg, state)
-
-        # Get the configuration for the SP.
-        if sp_entity_id in self.config.keys():
-            config = self.config[sp_entity_id]
-        else:
-            config = self.config["default"]
-
-        msg = "Using config {}".format(self._filter_config(config))
-        satosa_logging(logger, logging.DEBUG, msg, state)
+        issuer = data.auth_info.issuer
+        requester = data.requester
+        config = self.config.get(requester) or self.config["default"]
+        msg = {
+            "message": "entityID for the involved entities",
+            "requester": requester,
+            "issuer": issuer,
+            "config": self._filter_config(config),
+        }
+        satosa_logging(logger, logging.DEBUG, msg, context.state)
 
         # Ignore this SP entirely if so configured.
         if config["ignore"]:
-            msg = "Ignoring SP {}".format(sp_entity_id)
-            satosa_logging(logger, logging.INFO, msg, state)
+            msg = "Ignoring SP {}".format(requester)
+            satosa_logging(logger, logging.INFO, msg, context.state)
             return super().process(context, data)
 
         # The list of values for the LDAP search filters that will be tried in
         # order to find the LDAP directory record for the user.
-        filter_values = []
-
-        # Loop over the configured list of identifiers from the IdP to consider
-        # and find asserted values to construct the ordered list of values for
-        # the LDAP search filters.
-        for candidate in config["ordered_identifier_candidates"]:
-            value = self._construct_filter_value(
-                candidate,
-                data.subject_id,
-                data.subject_type,
-                data.auth_info.issuer,
-                data.attriutes,
-            )
-
+        filter_values = [
+            filter_value
+            for candidate in config["ordered_identifier_candidates"]
+            # Consider and find asserted values to construct the ordered list
+            # of values for the LDAP search filters.
+            for filter_value in [
+                self._construct_filter_value(
+                    candidate,
+                    data.subject_id,
+                    data.subject_type,
+                    issuer,
+                    data.attriutes,
```

Comment:
attributes instead of attriutes

Reply:
Your comment overlapped with me finding it and fixing it in my latest commit.